packages/internals/src/cli/checkUnsupportedDataProxy.ts

```diff
@@ -62,8 +62,9 @@ async function checkUnsupportedDataProxyMessage(command: string, args: Args, imp
 
       const datamodel = await fs.promises.readFile(argValue, 'utf-8')
       const config = await getConfig({ datamodel, ignoreEnvVarErrors: true })
-      const urlFromValue = config.datasources[0]?.url.value
-      const urlEnvVarName = config.datasources[0]?.url.fromEnvVar
+      const url = getEffectiveUrl(config.datasources[0])
+      const urlFromValue = url.value
+      const urlEnvVarName = url.fromEnvVar
```

Comment:
Nit:

```suggestion
      const { value: urlFromValue, fromEnvVar: urlFromEnvVarName } = getEffectiveUrl(config.datasources[0])
```

Reply:
since it's only a nit Let's merge without this change, once tests are passing